gnome-initial-setup/pages/account/gis-account-page.c

```diff
@@ -1218,6 +1226,34 @@ gis_account_page_save_data (GisPage *gis_page)
 }
 
 static void
+load_css_overrides (GisAccountPage *self)
+{
+  GtkCssProvider *provider;
+  GError *error;
+  GFile *file;
+
+  error = NULL;
+  provider = gtk_css_provider_new ();
+  file = g_file_new_for_uri ("resource:///org/gnome/initial-setup/account.css");
+  gtk_css_provider_load_from_file (provider, file, &error);
```

Comment:
You can use `gtk_css_provider_load_from_resource()` and avoid going through `GFile`.